### PR TITLE
Feature/hardware specs update

### PR DESCRIPTION
**Pull request by mfahampshire**
Re-org-ed the mix node page 
* made prep work with wallet and IPv6 speaking VPS more obvious + hardware specs
* bit of general organisation into different larger categories for easier searching
* removed a few caution notes, decluttered  

Fixed link that docusaurus was messing with automatically, Discord invite link now works again

### CODE REVIEW

File: docs/developers/getting-started.md [hunk 1]
```diff
@@ -37,4 +37,4 @@ Once you've decided which part of the stack best serves your aims, you have to d
 If you're wanting to develop on top of the infrastructure code itself and start adding additional features to existing nodes, check out the [Nym github repository](https://github.com/nymtech/nym). If you want to get to grips with running a node before starting this, check out the [Node Operators](/docs/next/run-nym-nodes/pre-built-binaries) section of these docs for more.  
 
 ## Integrations with existing apps 
-If you're building an application that you think could benefit from utilising the Nym network, get in touch via [max@nymtech.net](mailto:max@nymtech.net) or via [Discord](Discord.gg/nym) or Keybase (search for the `nymtech.friends` teams).  
+If you're building an application that you think could benefit from utilising the Nym network, get in touch via [max@nymtech.net](mailto:max@nymtech.net) or via [Discord](https://discord.com/invite/nym) or Keybase (search for the `nymtech.friends` teams).
```
Comment (iamonuwa) [inline docs/developers/getting-started.md]: We could also add `nymtech.friends` keybase URL here https://keybase.io/team/nymtech.friends

Comment (mfahampshire) [reply docs/developers/getting-started.md]: fixed in commit [1175f3e](https://github.com/nymtech/docs/pull/169/commits/1175f3e77fba671458527e594099b9f3c1aa3bd9)

File: docs/run-nym-nodes/nodes/mixnodes.md [hunk 2]
```diff
@@ -28,6 +34,21 @@ Remember that you can **only** use Cosmos `NYM` tokens to bond your mixnode. You
 #### Sandbox testnet
 Make sure to download a wallet and create an account as outlined above. Then head to our [token faucet](https://faucet.nymtech.net/) and get some tokens to use to bond it. 
 
+### VPS Hardware Specs
+You will need to requisition a VPS to run your mix node on. One key reason for this is that your node **must be able to send TCP data using both IPv4 and IPv6** (as other nodes you talk to may use either protocol.
```
Comment (iamonuwa) [inline docs/run-nym-nodes/nodes/mixnodes.md]: I think own is a better word than requisition.

File: docs/run-nym-nodes/nodes/mixnodes.md [hunk 3]
```diff
@@ -142,14 +163,16 @@ Initalise your mixnode with the following command, replacing the value of `--id`
...
+Your node **must** be able to send TCP data using **both** IPv4 and IPv6 (as other nodes you talk to may use either protocol).
 :::
 
-Your `--host` must be publicly routable on the internet in order to mix packets, and can be either an Ipv4 or IPv6 address. Your node _must_ be able to send TCP data using _both_ IPv4 and IPv6 (as other nodes you talk to may use either protocol). The `$(curl ifconfig.me)` command above returns your IP automatically using an external service. Alternatively, you can enter your IP manually wish. If you do this, remember to enter your IP **without** any port information.
+The `$(curl ifconfig.me)` command above returns your IP automatically using an external service. Alternatively, you can enter your IP manually wish. If you do this, remember to enter your IP **without** any port information.
 
-:::caution
-Please note that the `init` command will refuse to destroy existing mixnode keys.
+:::note
+The `init` command will refuse to destroy existing mix node keys.
```
Comment (iamonuwa) [inline docs/run-nym-nodes/nodes/mixnodes.md:175]: I noticed you changed all mixnode to mix node

**Review by iamonuwa — APPROVED** [review]
LGTM!

File: docs/run-nym-nodes/nodes/mixnodes.md (hunk 2 above)
Comment (mfahampshire) [reply docs/run-nym-nodes/nodes/mixnodes.md]: Good point

File: docs/run-nym-nodes/nodes/mixnodes.md (hunk 3 above)
Comment (mfahampshire) [reply docs/run-nym-nodes/nodes/mixnodes.md:175]: yeah just the ones not in code blocks, to get docs in line with the blogposts

**Pull request merged**